src/readthedocs_assistant/commands/migrate_config.py

```diff
@@ -93,7 +143,21 @@ async def fork_and_update(
         compare_url["html_url"],
     )
 
-    # TODO: Create pull request with message
+    accept = input("Continue? (yes/[no]) ") if interactive else "yes"
+    if accept == "yes":
+        pull_request = await gh.post(
+            f"/repos/{forked_repo['full_name']}/pulls",
+            data={
+                "title": "Migrate RTD conf",
```

Comment:
```suggestion
                "title": "Migrate Read the Docs configuration",
```

also, maybe "Update" or "Improve" instead of "Migrate"?

Reply:
Totally 👍🏽